sidebarsAstro.js

```diff
@@ -204,6 +201,13 @@ module.exports = {
             },
           ],
         },
+        {
+          type: "category",
+          label: "Organizations",
```

Comment:
```suggestion
          label: "Organization",
```
I'm fine either way but I feel like the vast vast majority of users will only ever have one Org, so it's okay to refer to it like that